api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
-
+from django.utils import timezone
 # Create your models here.
+
+class Task (models.Model):
+
+	STATUS_CHOICES = (
+		('UNASSIGNED', 'Unassigned'),
+		('WORKINGON', 'Working On'),
+		('COMPLETED', 'Completed'),
+		)
+
+	title = models.CharField(max_length=255, default=None)
+	description = models.TextField(max_length=1000)
+	status = models.CharField(max_length=15, choices=STATUS_CHOICES, default='UNASSIGNED')
```

Comment:
The value of default is with quotes if it's a string value, otherwise is without